container/pull.bzl

```diff
@@ -80,24 +80,31 @@ _container_pull_attrs = {
         doc = "(optional) The tag of the image, default to 'latest' " +
               "if this and 'digest' remain unspecified.",
     ),
+    "import_tags": attr.string_list(
```

Comment:
These must be in alphabetical order (name is fine btw)

Reply:
Fixed.